src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/RetrieveMinimalOrganisationsInfoTest.java

```diff
@@ -0,0 +1,264 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.professionalapi.controller.constants.ErrorConstants.ACCESS_EXCEPTION;
+import static uk.gov.hmcts.reform.professionalapi.domain.OrganisationStatus.ACTIVE;
+import static uk.gov.hmcts.reform.professionalapi.helper.OrganisationFixtures.someMinimalOrganisationRequest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import net.serenitybdd.junit.spring.integration.SpringIntegrationSerenityRunner;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.util.CollectionUtils;
+import uk.gov.hmcts.reform.professionalapi.controller.advice.ErrorResponse;
+import uk.gov.hmcts.reform.professionalapi.controller.constants.IdamStatus;
+import uk.gov.hmcts.reform.professionalapi.controller.request.NewUserCreationRequest;
+import uk.gov.hmcts.reform.professionalapi.controller.request.OrganisationCreationRequest;
+import uk.gov.hmcts.reform.professionalapi.controller.response.OrganisationMinimalInfoResponse;
+import uk.gov.hmcts.reform.professionalapi.domain.ContactInformation;
+import uk.gov.hmcts.reform.professionalapi.domain.Organisation;
+import uk.gov.hmcts.reform.professionalapi.domain.OrganisationStatus;
+import uk.gov.hmcts.reform.professionalapi.idam.IdamOpenIdClient;
+
+@RunWith(SpringIntegrationSerenityRunner.class)
+public class RetrieveMinimalOrganisationsInfoTest extends AuthorizationFunctionalTest {
+
+    @Value("${activeOrgsExternalEnabled}")
+    private boolean activeOrgsExternalEnabled;
+
+    private static final String STATUS_PARAM_INVALID_MESSAGE =
+            "Please check status param passed as this is invalid status.";
+
+    private List<OrganisationMinimalInfoResponse> activeOrgs = new ArrayList<>();
+    private List<OrganisationMinimalInfoResponse> pendingOrgs = new ArrayList<>();
+    private List<OrganisationMinimalInfoResponse> noAddressOrgs = new ArrayList<>();
+
+    private OrganisationMinimalInfoResponse organisationEntityResponse1;
+    private OrganisationMinimalInfoResponse organisationEntityResponse2;
+    private OrganisationMinimalInfoResponse organisationEntityResponse3;
+    private OrganisationMinimalInfoResponse organisationEntityResponse4;
+    private OrganisationMinimalInfoResponse organisationEntityResponse5;
+    private OrganisationMinimalInfoResponse organisationEntityResponse6;
+    private String orgIdentifier1;
+    private String orgIdentifier2;
+    private String orgIdentifier3;
+    private String orgIdentifier4;
+    private String orgIdentifier5;
+    private String orgIdentifier6;
+    private List<String> validRoles = new ArrayList<>();
+    private List<String> orgResponseInfo = new ArrayList<>();
+
+
+    @Test
+    @SuppressWarnings("unchecked")
+    //AC:1
+    public void should_retrieve_organisations_info_with_200_with_correct_roles_and_status_active() {
```

Comment:
If you are deploying this endpoint in production with toggle OFF then you should have only 1 test case enabled with happy path gives back 403.
When Toggle ON will be deployed then you need to delete this test case and enable others.

Reply:
Added here: https://github.com/hmcts/rd-professional-api/pull/694/commits/43fb9fec023e5b06187c21c4f30b0ec5ef08adcd

Reply:
test case need not be deleted.. it can be just toggled off..you remove the test cases only when the toggle is removed

Reply:
Sure, I have kept the other (toggled on) test cases and included a `403` test when it is toggled off: https://github.com/hmcts/rd-professional-api/commit/43fb9fec023e5b06187c21c4f30b0ec5ef08adcd